### DISCUSSION TITLE
Can I use gridjs without mermaid.min.css ?

### DISCUSSION THREAD

**Discussion by leodip** [grid-js/gridjs#1395] [Q&A]
Hi,
I'm using TailwindCss in my application.

I would like to use gridjs with class names from Tailwind to style my grid. 

It's easier for me if I just remove the declaration of mermaid.min.css from my HTML.

On a first look, the grid seems to work fine without it...

But I wanted to confirm here to be sure -  is it ok if I remove the CSS file (mermaid.min.css)? Or does gridjs depend strongly on it?

Thanks

**Comment by Nemerion** [comment]
Hey, I recently started to use Gridjs, and found that they put the source mermaid styles inside the node modules folder.
So for me, I just copied it entirely, created another 'grid_styles.css', paste it and modified the parts that I needed to match my style criteria.

But to be honest, you actually can avoid using the Mermaid option, but you will have no styles at all (it's on you to create your styles :)).
